security/rs-security-best-practices.md

```diff
@@ -0,0 +1,56 @@
+
+## Configuration Settings
+
+The suggestions in this section are the responsibility of the administrator deploying the Telerik Report Server. They are general security settings of web applications deployed on the Windows IIS Servers. We have implemented most of them and let you switch them as configuration options when installing the Report Server:
+
+* Configure your Report Server instance to run under [HTTPS protocol](https://developer.mozilla.org/en-US/docs/Glossary/HTTPS). Consider the advice in the article [Configuring IIS Website to Work Over HTTPS]({%slug configuring-iis-website-to-work-over-https%}).
+* Let the user with lowered permissions in IIS and Report Server ServiceAgent be your preferred choice. By default, the MSI installer suggests applying the [principle of least privilege](https://learn.microsoft.com/en-us/entra/identity-platform/secure-least-privileged-access) and creating a dedicated Windows user named __ReportServerUser__ whose identity will be used by both applications. The user is granted the minimum necessary permissions to operate within the installation folder of Telerik Report Server as explained in [Report Server Installation]({%slug installation%}). 
+* Disable the [CORS]({%slug cors%}) if possible; or enable CORS only for trusted hosts.
+* Use the [Encryption]({%slug security%}#encryption) functionality of the Report Server to keep your assets safer.
+* Consider the [Rate Limiting]({%slug security%}#rate-limiter) in your Report Server to restrict the network traffic and prevent bad agents from exhausting system resources.
+
+## Working with Report Server Assets
+
+After successfully installing and configuring the Telerik Report Server, the user with sufficient rights, for example, the admin user and those with granted permissions for the corresponding resources, may further enhance its security through the administration policy options provided by the server. Check the article section [Manage Permissions]({%slug users%}#manage-permissions) for the available access modes, scopes, and targets you may provide to Report Server Users. The built-in user roles are listed in the article [User Roles]({%slug user-roles%}).
+
+The admin user and the users with sufficient permissions may control access to all or part of the following Report Server resources:
+
+* [Manage User Permissions]({%slug users%}#manage-user-permissions)
+  Each Report Server User is responsible for the Resources in his/her control. 
+* [Reports Management]({%slug reports-management%})
+  Grant users access only to the Reports/Categories they need. Ensure they are entitled to access the corresponding data sources.
+* [Data Connections Management]({%slug data-connections-management%})
+  Ensure the Data Connections do not contain credentials. The identity used to connect to the database or web service should be with limited permissions.
```

Comment:
Not sure if the "ENSURING" can be achieved. If we're using Integrated Security, it's highly likely to have higher credentials than a dedicated MSSQL user with lower permissions. 
I would phrase it like this:
"Try applying the principle of least privilege to the credentials provided in connection strings in Data Connections view. The identity used to connect to the database or the web service should be with the lowest permissions possible. If possible, try using a dedicated user account with read-only permissions and access scope including only the tables needed for reporting purposes."